iothub/device/src/IotHubClientOptions.cs

```diff
@@ -67,30 +67,6 @@ public IotHubClientOptions(IotHubClientTransportSettings transportSettings)
         /// </remarks>
         public SdkAssignsMessageId SdkAssignsMessageId { get; set; } = SdkAssignsMessageId.Never;
 
-        /// <summary>
-        /// The suggested time to live value for tokens generated for SAS authenticated clients.
-        /// The <see cref="TimeSpan"/> provided should be a positive value, signifying that it is not possible to generate tokens that have already expired.
-        /// If unset the generated SAS tokens will be valid for 1 hour.
-        /// </summary>
-        /// <remarks>
-        /// This is used only for SAS token authenticated clients through either the
-        /// <see cref="IotHubDeviceClient.CreateFromConnectionString(string, IotHubClientOptions)"/> flow, the <see cref="IotHubModuleClient.CreateFromConnectionString(string, IotHubClientOptions)"/> flow
-        /// or the <see cref="IotHubModuleClient.CreateFromEnvironmentAsync(IotHubClientOptions)"/> flow.
-        /// </remarks>
-        public TimeSpan SasTokenTimeToLive { get; set; }
```

Comment:
The connection string flow has an associated IAuthenticationMethod that can be used for this feature.